content/webapp/pages/exhibition-guide.tsx

```diff
@@ -322,6 +370,13 @@ const ExhibitionLinks: FC<ExhibitionLinksProps> = ({ stops, pathname }) => {
           title="Listen, without audio descriptions"
           text="Find out more about the exhibition with short audio tracks."
           color="newPaletteOrange"
+          onClick={event => {
+            event.stopPropagation();
```

Comment:
what would this propagate to that we want to prevent?

Reply:
You're right, I don't think there is any event I want to stop from propagation